doc/rados/operations/devices.rst

```diff
@@ -138,71 +142,75 @@ for a specific timestamp) with:
 Failure prediction
 ------------------
 
-Ceph can predict life expectancy and device failures based on the
-health metrics it collects.  There are three modes:
+Ceph can predict life expectancy and device failures by analyzing the health
```

Comment:
can predict drive life expectancy and incipient failures by

Reply:
Damn it. I meant to put in "drive life expectancy". The way it's written now, it reads as though Ceph contains Dr. Hugo Pinero's invention from Robert Heinlein's short story "Life-Line".